xapian-core/tests/unittest.cc

```diff
@@ -680,6 +682,86 @@ static bool test_uuid1()
     return true;
 }
 
+// Classes used by movesupport1 test
+class A : public Xapian::Internal::intrusive_base {
+    int x = 0;
+  public:
+    A(int x_) : x(x_) {}
```

Comment:
Unless you want the compiler to implicitly convert `int` to `A`, it's a good idea to mark constructors which take a single parameter (either explicitly, or via default parameter values) as `explicit`.  Similarly for `B` below.

It's not an issue as the code is now, but if we're consistent about it we're more likely to remember to do it when it matters, and also won't get bitten by it suddenly starting to matter as the code evolves.

Reply:
I agree. Updated. Since B takes two arguments now, explicit keyword not added.